themes/hugo-rladies/layouts/blog/single.html

```diff
@@ -1,45 +1,72 @@
 
-{{ define "main"}}
-    <div class="entry-content">
-      <p>{{ .Content }}</p>
-    </div>
-
-    {{ with .Params.tags -}}
-    
-      <div class="post-tags my-2">
-        {{ i18n "tags"}}:
-        {{ range . }}
-        {{- $name := . -}}
-        {{- with $.Site.GetPage "taxonomy" (printf "tags/%s" $name) | default ($.Site.GetPage "taxonomy" (printf "tags/%s" ($name | urlize))) -}}
-        <a href="{{ .Permalink }}">{{ $name }}</a>
-        {{ end -}}
-        {{ end }}
-      </div>
-    {{- end }}
+{{ define "title" }}
+<header class="entry-header">
+  <div class="entry-meta">
+    <span class="posted-on">
+      {{if isset .Params "date"}}
+        <time class="entry-date published" datetime="{{ .Date.Format .Site.Params.DateForm }}">
+          {{- .Date.Format "January 2, 2006" -}}
+        </time>
+      {{end}}
+    </span>
+  </div>
+  <h1 class="section-title">{{ .Title }}</h1>
+  <div class="container">
+    <div class="row gx-5 align-items-center">
+      {{ with .Params.image }}
+        <div class="col-12 col-md-4">
+          <img src="{{ .path }}" alt="{{ .alt | default $.Title }}" class="img-fluid">
+        </div>
+      {{ end }}
 
-    <!-- Author Info -->
-     <div class="my-2">
-       {{ $type := (printf "%T" .Params.author) }}
-      
-      {{ if eq $type "string" }}
-         {{ i18n "author" }}: {{ .Params.author | markdownify }}
-      {{ else }}
-        {{ with .Params.author }}
-          {{ i18n "authors" }}: 
-          {{ range . }}
-            <li>{{. | markdownify }}</li>
+      <div class="col">
+        <div class="row gx-5">
+          {{ with .Params.author }}
+            <div id="authors" class="col-12 col-md-6">
+              <h5 class="fw-bold my-3">
+                {{ i18n "authors" }}
```

Comment:
add some pluralization logic to this and editor and tag?

Reply:
I'm not entirely sure how pluralization works in terms of multiple language support. But our i18n does have dictionary for both "author" and "authors" (same for editor(s)". If switching between those two sounds like a good idea?